internal/staticupload/staticupload.go

```diff
@@ -57,6 +62,14 @@ type Config struct {
 	CacheInvalidationWaitTimeout time.Duration
 }
 
+// Validate checks if all necessary values are set and sets default values otherwise.
+func (c *Config) Validate() error {
```

Comment:
Is this method required to implement an interface?
If not you can remove the return value, as this will always return `nil`